src/go/k8s/cmd/configurator/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Build the manager binary
+FROM golang:1.15 as builder
+
+# Copy the rpk as a close depedency
+WORKDIR /workspace
+COPY rpk/ rpk/
```

Comment:
This could potentially create inconsistencies between the redpanda version used and rpk's code

Reply:
oh yeah, shouldn't we depend on RPK as a module or is that problematic?

Reply:
I agree, but `configurator` uses only config package. So to not maintain multiple version of the config I include this dependency.

Reply:
We did it back at the first operator (different repo). I can not find the comment that @0xdiba made, but it was in something about introducing braking change in whole mono repo. If we have module dependency that will be not possible.

Reply:
what worries me in this case would be the configurator having newer code than the redpanda image itself, leading to configuration instabilities, crashes etc :thinking: